### DISCUSSION TITLE
Buttons not working

### DISCUSSION THREAD

**Discussion by evetra** [knowledgefutures/pubpub#2894] [Support]
None of the buttons seem to be functioning, whether it is adding members or importing a file. Something seems to be quite broken. I am on a Mac using Chrome. Is this software not supported for Chrome?

**Comment by gabestein** [comment]
I am unable to produce this issue on a Mac with the latest version of Chrome. Is it possible that you have a plugin installed that is interfering with the functioning of the site?

If you can describe the issue more, including the Mac OS X and Chrome version, and send a screen capture showing the issue, we may be able to help more.
